content/blog/2020/11/2020-11-08-help-test-jenkins.adoc

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Help Us Test Jenkins"
+tags:
+- jenkins
+- testing
+- ui
+author: markewaite
+description: "Latest Jenkins weekly releases contain large changes.  Help us test the most recent releases"
+opengraph:
+  image: /images/post-images/2020-11-08-help-test-jenkins/help-test-jenkins.png
+---
+
+Beginning with the Jenkins 2.264 release on October 27, 2020, we've entered a period where the Jenkins weekly releases will include more significant changes than usual.
+That period of more significant changes is expected to continue for a month or more.
+
+We're excited for the changes and invite you to join us as we test those changes.
+This is an especially valuable time for users and administrators to test the weekly releases and report issues with them.
+
+The changes include:
+
+* <<UI - Tables to Divs>>
+* <<Core - Spring Security replaces Acegi Security>>
+* <<Core - XStream unfork>>
+* <<UI - JQuery upgrade>>
+
+== UI - Tables to Divs
+
+Jenkins 2.264 is the first weekly release to include the "Tables to Divs" user interface work of link:https://github.com/jsoref/[Josh Soref], link:https://github.com/timja/[Tim Jacomb], and https://github.com/fqueiruga[Felix Queiruga].
+It is a significant step to improve the Jenkins user interface, especially for users on narrow devices like tablets and phones.
+
+=== A better user interface
+
+The transition from using HTML `table` tags to using HTML `div` tags provides a more attractive user interface for all users and a much better experience for users on narrower devices.
+Before the conversion from table tags to div tags, the **"Manage Jenkins"** page looked like this in a 1024x768 window:
+
+image:/images/post-images/2020-11-08-help-test-jenkins/ui-using-tables.png[Before the pull request]
+
+After the conversion, the **"Manage Jenkins"** page now looks like this:
+
+image:/images/post-images/2020-11-08-help-test-jenkins/ui-using-divs.png[After the pull request]
+
+The user interface improvements from the transition are a nice step forward for Jenkins.
+However, because the user interface improvements require changes in plugins, we need your help.
+
+We need users to test the latest weekly Jenkins releases with the plugins and configurations that are most important to them.
+When users detect an issue, we need them to link:/participate/report-issue/[report the issue] with enough detail that a plugin maintainer can fix the issue.
+
+=== Plugin developers
+
+In addition to testing, we've already identified several plugins that may need changes.
+See the link:https://issues.jenkins-ci.org/browse/JENKINS-62437[Jira epic] for plugins that are likely to need changes for the new user interface layout.
+
+If you can assist with plugin testing and code changes, select one of the plugins from that epic, test it, and propose a pull request to help with this user interface transition.
```

Comment:
migration guide should be mentioned https://www.jenkins.io/doc/developer/views/table-to-div-migration/

Reply:
Added a migration guide paragraph in 2aa7e04126470a7464c732fae8af214f5dd3afa2